tests/wicked/startandstop/ref/t01_standalone_card_ifdown_ifreload.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Startandstop test cases for wicked
+# Standalone card - ifdown, ifreload
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base 'wickedbase';
+use strict;
+use testapi;
+use utils 'systemctl';
+use lockapi;
+use mmapi;
```

Comment:
Please only includ the actually used packages, applies to all files

Reply:
sure, no need for mmapi here, thanks